### DISCUSSION TITLE
Set max items per minute?

### DISCUSSION THREAD

**Discussion by PelleHanses** [excalidraw/excalidraw#8447] [General]
I wonder if there is any way to limit the number of elements that a user can insert into a board?
We host our own instance of Excalidraw and have it linked to Jitsi. We have discovered that it is possible to destroy the experience for everyone if a user creates or copiy/paste too many elements. It is not a big problem now but it would be good if it was possible to counteract in some way. Maybe it would be best if you can put the maximum number of elements created per minute or per user.

**Comment by Mrazator** [comment]
There is no hard limit on a number of elements, it all depends on the specific browser canvas size limits and the client-side hardware.  However, on excalidraw.com we have an indirect 5MB limit for the whole scene.
